Marlin/ubl_motion.cpp

```diff
@@ -250,9 +231,8 @@
     const float m = dy / dx,
                 c = start[Y_AXIS] - m * start[X_AXIS];
 
-    const bool inf_normalized_flag = isinf(e_normalized_dist),
-               inf_m_flag = isinf(m);
-
+    const bool inf_normalized_flag=isinf(e_normalized_dist),
+               inf_m_flag=isinf(m);
```

Comment:
Yes.   But if a variable is being declared and it is being assigned a value in the declaration, that is a 'pre-initialized variable'.    Especially in the case of globals that are being given constants...  no code has to execute.  It just has the right value at the very start of the program because its space in memory was 'pre-initialized' with the correct value.